vuu-ui/packages/vuu-data/src/array-data-source/array-data-source.ts

```diff
@@ -1,6 +1,6 @@
 import { DataSourceFilter, DataSourceRow } from "@finos/vuu-data-types";
```

Comment:
doesn't stop us merging, but we're still lacking a `set config` that will allow us to set multiple attributes at same time. Will be needed for full compatability with RemoteDataSource

Reply:
Will quickly add it in in the coming week! I shifted off that after discussion with Steve that it would be more modular with the current setup.